kneipentour/index.php

```diff
@@ -34,9 +35,15 @@
             echo ($h['error'] == '' ? '' : "<div class='block info'>{$h['error']}</div>");
         ?>
         <div class="block>">
-            <form method="post" action="#">
+            <!--<form method="post" action="#">
                 <input type="submit" value="Anmelden" <?php echo $h['enabled'] ? '' : 'disabled' ?>></td></tr>
-            </form>
+            </form>-->
```

Comment:
This as well? And if so also in the next two and last three files listed in the diff (seems like there is a bit of redundancy :P).